entname/entname.go

```diff
@@ -88,6 +118,57 @@ func AggregateMemberInterface(dp *DeviceParams, index int) (string, error) {
 	return namer.AggregateMemberInterface(index)
 }
 
+// Port returns the vendor-specific name of the physical interface with the
+// given port parameters.
+func Port(dp *DeviceParams, pp *PortParams) (string, error) {
+	namer, err := lookupNamer(dp)
+	if err != nil {
+		return "", err
+	}
+	npp, err := namerPortParams(pp, namer.IsFixedFormFactor())
+	if err != nil {
+		return "", err
+	}
+	return namer.Port(npp)
```

Comment:
I'm curious if you think the Namer interface needs to be exported considering we have helpers in `entname`? The namer implementation will also need to handle negative indices, just like here, right?